product-service/db/ddl.sql

```diff
@@ -0,0 +1,21 @@
+create EXTENSION if not EXISTS "uuid-ossp";
+
+
+DROP TABLE IF exists products CASCADE; 
+
+create table products (
+	id uuid primary key default uuid_generate_v4(),
+	title text NOT NULL,
+	description text,
+	price integer
+);
+
+DROP TABLE IF exists stocks;
+
+create table stocks (
+	product_id uuid unique,
```

Comment:
I'm not sure, but it seems, that this field should always be filled. But right now, there is no 'not null', so we can set it with 'null'. PostgreSQL doesn't restrict to paste 'null' in columns with 'unique'. Quatation from documentation 

> For the purpose of a unique constraint, null values are not considered equal.

Reply:
Fixed [973bc42](https://github.com/YakovGrischenko/nodejs-aws-be/pull/2/commits/973bc42a0f7e8ccd468c08c48e345cb930a096e0)

Reply:
I have checked on my DB, in this case 'uuid' type doesn't allow to set 'null' in this column, because 'null' is invalid 'uuid'. But, I think, in other cases this information about 'unique' will be more useful :)